Source/WebKit/UIProcess/API/C/WKBackForwardListItemRef.cpp

```diff
@@ -41,6 +41,7 @@ WKURLRef WKBackForwardListItemCopyURL(WKBackForwardListItemRef itemRef)
     return WebKit::toCopiedURLAPI(toImpl(itemRef)->url());
 }
 
+// This function is deprecated and now always returns the empty string.
 WKStringRef WKBackForwardListItemCopyTitle(WKBackForwardListItemRef itemRef)
 {
     return WebKit::toCopiedAPI(toImpl(itemRef)->title());
```

Comment:
There's no need to leave comments saying the API is deprecated. Those have no effect; nobody will ever notice them. Instead, use `WK_C_API_DEPRECATED` on the declaration of this function in the header file, which is what will trigger `-Wdeprecated-declaration`.

Please also change it to just directly return the empty string. The goal is to comprehensively remove the title property wherever possible (everywhere except where it is exposed to the web or exposed in API).

Reply:
```cpp
WKStringRef WKBackForwardListItemCopyTitle(WKBackForwardListItemRef itemRef)
{
    return WebKit::toCopiedAPI(toImpl(itemRef)->title());
``` 
> Please also change it to just directly return the empty string.

I changed it to `return 0` — since the return type of the function is `WKStringRef`. Or do you mean I should also change the return type of the function? (so that I can make it return the empty string)

Reply:
No, certainly don't change the return type of the function.

I think this function should return a non-NULL/nonzero WKStringRef that contains an empty string.

Reply:
Actually I see Chris is saying this is fine if changed to `nullptr`. He has more experience with these C APIs than I do, so let's trust his answer instead.

Reply:
I actually like Michael's idea better. Let's return an empty string here. It is less likely to cause backward compatibility issues. Thanks for catching it @mcatanzaro !